string id # The ID of the rule to delete.
---
